javav2/usecases/creating_fsa_app/src/main/java/com/example/fsa/handlers/S3Handler.java

```diff
@@ -0,0 +1,26 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.fsa.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.example.fsa.services.ExtractTextService;
+import java.util.Map;
+
+public class S3Handler implements RequestHandler<Map<String, Object>, String>{
```

Comment:
S3Handler seems like a misnomer.

Reply:
Not sure what you mean by "just an example". That's what we do.

Reply:
Right. When a customer goes to find the "ExtractText" handler, they're not going to see one. They're just going to see S3Handler. They'll have to look at the package name in the functions.ts file to track it down. Also, S3Handler implies that we're handling S3, which isn't the most accurate way of thinking about it.